backend/api/publishing.py

```
@@ -850,6 +2010,29 @@ async def cleanup_archives(
         "records_cleaned": cleaned_count
     }
 
+@router.post("/cleanup/temp-pdfs")
+async def cleanup_temp_pdfs(
+    days_old: int = Query(7, description="Delete files older than this many days"),
+    db: Session = Depends(get_db),
```

Ruff (0.14.3)

Unused function argument: `db`

(ARG001)